src/Stickiness/StickinessCalculator.php

```diff
@@ -4,5 +4,5 @@
 
 interface StickinessCalculator
 {
-    public function calculate(string $id, string $groupId, int $normalizer = 100): int;
+    public function calculate(string $id, string $groupId, int $normalizer = 100, int $seed = 0): int;
```

Comment:
This would need a major version release as it's a BC break.

Reply:
Yes, yes it will. We think that's correct anyway since this will trigger a new distribution of variants and that's likely to be a significant change for anyone using the SDK

So probably time to discuss that new major version then

Reply:
Definitely, there are a few changes racked up over the SDK's life that are just waiting for a new major release.